packages/ui/src/lib/table/Table.svelte

```diff
@@ -150,7 +157,7 @@ function setGridColumns() {
               ? 'justify-self-center'
               : 'justify-self-start'} self-center select-none"
           class:cursor-pointer="{column.info.comparator}"
-          on:click="{() => sort(column)}"
+          on:click="{sort.bind(undefined, column)}"
```

Comment:
I don't love this... but don't have any idea on what else to do - not worth holding up for.

Reply:
yes :-/ either we disable the lint rule or we come with another idea